index.js

```diff
@@ -0,0 +1,120 @@
+const allTripsUrl = " https://ada-backtrek-api.herokuapp.com/trips";
+
+const singleTripUrl = "https://ada-backtrek-api.herokuapp.com/trips/";
+
+const reserveUrl = "https://ada-backtrek-api.herokuapp.com/trips/1/reservations";
+
+const reportStatus = (message) => {
+  $('#status-message').html(message);
+};
+
+const reportError = (message, errors) => {
+  let content = `<p>${message}</p>`
+  content += "<ul>";
+  for (const field in errors) {
+    for (const problem of errors[field]) {
+      content += `<li>${field}: ${problem}</li>`;
+    }
+  }
+  content += "</ul>";
+  reportStatus(content);
+};
+
+const loadTrips = function loadTrips() {
+  let tripList = $('#trip-list');
+
+  axios.get(allTripsUrl)
+  .then((response) => {
+    response.data.forEach((trip) => {
+      console.log(trip);
+      tripList.append(`<tr><td><a class="trip" data-id="${trip.id}" href="#">${trip.name}</a></td></tr>`);
+    });
+    reportStatus('Trips Loaded!');
+  })
+  .catch((error) => {
+    console.log(error);
+    if (error.response.data && error.response.data.errors) {
+      // User our new helper method
+      reportError(
+        `Something's wrong: ${error.message}`,
+        error.response.data.errors
+      );
+    } else {
+      reportStatus(`Error: ${error.message }`);
+    }
+  });
+};
+
+const getTrip = function getTrip(event) {
+  event.preventDefault();
+
+  let display = $('#new-reserve');
+  display.show();
+
+
+  axios.get(singleTripUrl + $(this).data("id"))
+  .then((response) => {
+    $('#details').empty().append(`<h2>${response.data.name}</h2><h3>${response.data.continent}</h3><p>${response.data.about}</p><p>Category of trip: ${response.data.category}</p><p>Duration of trip (in weeks): ${response.data.weeks}</p><p>$${response.data.cost}</p>`);
+    reportStatus('Trips Loaded!');
```

Comment:
Line 57 should be broken up into multiple lines. 80 characters is usually considered the widest a line should be.

Also, it might make sense to break the code to render trip details out into a separate function - as is, this function is doing two things (making the request, and rendering data). See also http://callbackhell.com/